klaus/views.py

```diff
@@ -37,7 +37,8 @@ def repo_list():
     else:
         sort_key = lambda repo: (-(repo.get_last_updated_at() or -1), repo.name)
     repos = sorted(current_app.repos.values(), key=sort_key)
-    return render_template('repo_list.html', repos=repos, base_href=None)
+    invalid_repos = sorted([os.path.basename(path) for path in current_app.invalid_repos])
```

Comment:
See comment about `.name` above.

Maybe it's handy to use a `InvalidRepo` class that has a `.name` attribute. So `FancyRepo` for good repos, `InvalidRepo` for bad repos. Not sure.

Reply:
An `InvalidRepo` class seems coherent to me, especially if more information about invalid repos is stored in the future (the reason why it is invalid for instance). I will add a new class for them.